core/workspace_svg.js

```diff
@@ -978,17 +978,54 @@ Blockly.WorkspaceSvg.prototype.paste = function(xmlBlock) {
 };
 
 /**
+ * Refresh the toolbox unless there's a drag in progress.
+ */
+Blockly.WorkspaceSvg.prototype.refreshToolboxSelection = function() {
+  if (this.toolbox_ && this.toolbox_.flyout_ && !Blockly.Flyout.startFlyout_) {
```

Comment:
Blockly.Flyout.startFlyout_ shouldn't exist anymore.  (The old line 989 was a typo).  Check for this.currentGesture_ instead (see old line 1004).